src/backend/app.js

```diff
@@ -21,59 +21,59 @@ app.use(cors());
 
 router.use("/meals", mealsRouter);
 
-app.get('/future-meals', async (req, res) => {
-  try {
-    const meal = await knex.raw("SELECT * FROM meal WHERE `when` > now()");
-    res.status(200).json(meal[0]);
-  } catch (error) {
-    res.status(500).json({ error: 'An error occurred.'});
-  }
-});
-
-app.get('/past-meals', async (req, res) => {
-  try {
-    const meal = await knex.raw("SELECT * FROM meal WHERE `when` < now()");
-    res.status(200).json(meal[0]);
-  } catch (error) {
-    res.status(500).json({ error: 'An error occurred.'});
-  }
-});
-
-app.get('/all-meals', async (req, res) => {
-  try {
-    const meal = await knex.raw("SELECT * FROM meal ORDER BY id");
-    res.json(200).json(meal[0]);
-  } catch (error) {
-   res.status(404).json(error);
-  }
-});
-
-app.get('/first-meal', async (req, res) => {
-  try {
-    const meal = await knex.raw("SELECT * FROM meal ORDER BY id LIMIT 1");
-    if (meal.length > 0) {
-      res.status(200).json(meal[0]);
-    } else {
-      res.status(404).json("There are no meals.");
+ app.get('/future-meals', async (req, res) => {
+   try {
+     const meal = await knex.raw("SELECT * FROM meal WHERE `when` > now()");
+     res.status(200).json(meal[0]);
+   } catch (error) {
+     res.status(500).json({ error: 'An error occurred.'});
+   }
+ });
+ 
+ app.get('/past-meals', async (req, res) => {
+   try {
+     const meal = await knex.raw("SELECT * FROM meal WHERE `when` < now()");
+     res.status(200).json(meal[0]);
+   } catch (error) {
+     res.status(500).json({ error: 'An error occurred.'});
+   }
+ });
+ 
+ app.get('/all-meals', async (req, res) => {
+   try {
+     const meal = await knex.raw("SELECT * FROM meal ORDER BY id");
+     res.json(200).json(meal[0]);
+   } catch (error) {
+    res.status(404).json(error);
+   }
+ });
+ 
+ app.get('/first-meal', async (req, res) => {
+   try {
+     const meal = await knex.raw("SELECT * FROM meal ORDER BY id LIMIT 1");
```

Comment:
would be better to add ASC keyword to the query

Reply:
Ok, thank you for your review. I fix it now :)